core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionActor.scala

```diff
@@ -412,6 +413,10 @@ trait WorkflowSubmission extends FutureSupport with LazyLogging with MethodWiths
       val executionServiceKey = workflowSubmitResult._1
       val executionServiceResults = workflowSubmitResult._2
 
+      // Emit metric for time until Cromwell has processed workflow.
+      val elapsedTime = System.currentTimeMillis() - submissionRec.submissionDate.getTime
+      workflowToCromwellLatency.update(elapsedTime, TimeUnit.MILLISECONDS)
```

Comment:
Just to make sure I am understanding this correctly, this would be the time between when a "submission" is created and after all the workflows in submission have been submitted to Cromwell?

Reply:
The intent was to capture the latency between initial Rawls submission and workflow submission to Cromwell, for each workflow in the batch.